CHANGELOG.rst

```diff
@@ -2,10 +2,56 @@
 Release Notes
 =============
 
+.. _Release Notes_6.3.3:
+
+6.3.3
+=====
+2018-07-17
+
+.. _Release Notes_6.3.3_Prelude:
+
+Prelude
+-------
+
+Release on: 2018-07-17
+
+- Please refer to the `6.3.3 tag on integrations-core <https://github.com/DataDog/integrations-core/releases/tag/6.3.3>`_ for the list of changes on the Core Checks.
+
+- Please refer to the `6.3.3 tag on trace-agent <https://github.com/DataDog/datadog-trace-agent/releases/tag/6.3.3>`_ for the list of changes on the Trace Agent.
+
+- Please refer to the `6.3.3 tag on process-agent <https://github.com/DataDog/datadog-process-agent/releases/tag/6.3.3>`_ for the list of changes on the Process Agent.
+
+
+.. _Release Notes_6.3.3_Enhancements Notes:
+
+Enhancements Notes
```

Comment:
this title is weird, `Enhancements` is enough I'd say :)

(probably needs to be fixed in reno's config too, feel free to do it in a different PR)